go.mod

```diff
@@ -84,6 +85,28 @@ require (
 	google.golang.org/grpc/examples v0.0.0-20210430044426-28078834f35b
 	google.golang.org/protobuf v1.34.1
 	gopkg.in/DataDog/dd-trace-go.v1 v1.64.0
+=======
+	go.etcd.io/etcd/api/v3 v3.5.18
+	go.etcd.io/etcd/client/pkg/v3 v3.5.18
+	go.etcd.io/etcd/client/v3 v3.5.18
+	go.uber.org/mock v0.5.0
+	golang.org/x/crypto v0.35.0 // indirect
+	golang.org/x/mod v0.23.0 // indirect
+	golang.org/x/net v0.36.0
```

Comment:
So this ends up needing Go 1.23.0 now, which is a problem here. See https://github.com/golang/net/commit/5095d0cf1463414ad99ced9d5032eae6175f5ac5